batch2/batch/driver/templates/index.html

```diff
@@ -40,6 +40,7 @@ <h1>Instances</h1>
     </thead>
     <tbody>
       {% for instance in instances %}
+      {% if instance.state != 'removed' %}
```

Comment:
We can't keep the removed instances in memory.  We should just keep the non-removed ones.